flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/batch/BatchExecExchange.scala

```diff
@@ -138,7 +138,7 @@ class BatchExecExchange(
     val conf = planner.getTableConfig
     val shuffleMode = getShuffleMode(conf.getConfiguration)
 
-    relDistribution.getType match {
+    ExecNode.setManagedMemoryWeight(relDistribution.getType match {
```

Comment:
do we have to set managed memory for partition transformation?

Reply:
CC: @zhuzhurk

Reply:
For the sake of insurance, set 0

Reply:
No need to do that since it does not make sense. No StreamNode will be generated from a PartitionTransformation.